assets/cubyz/tools/axe.zig.zon

```diff
@@ -1,132 +1,98 @@
 .{
 	.blockTags = .{.wood},
-	.slotTypes = .{
+	.slots = .{
+		.gemSlot, .backbone, .none,     .none,     .none,
+		.blode,   .handle,   .backbone, .none,     .none,
```

Comment:
Blonde slot still here? Do we keep named slots?

Reply:
oops, forgot to remove them